src/cli_plugin/list/list.js

```diff
@@ -1,13 +1,28 @@
-import { statSync, readdirSync } from 'fs';
+import { statSync, readdirSync, readFileSync, accessSync, R_OK } from 'fs';
 import { join } from 'path';
 
+function readable(path) {
```

Comment:
in this case it shouldn't be needed, readFileSync will already throw if the file isn't available.

thoughts on something like:
```javascript
try {
  var packageJSON = JSON.parse(readFileSync(packagePath, 'utf8'));
  logger.log(filename + '@' + packageJSON.version);
} catch(e) {
  logger.log(filename);
}
```